src/main/webapp/js/chartCode.js

```diff
@@ -1,26 +1,53 @@
-
 /* global google */
 
 // Load the Visualization API and the corechart package.
-google.load("visualization", "1", { packages: ["corechart"] });
+google.load("visualization", "1", {packages: ["corechart"]});
 google.setOnLoadCallback(drawChart);
 
 function handleQueryResponse(response) {
```

Comment:
@theider why is there still a handleQueryResponse? Seems like nothing is calling it.